### ISSUE TITLE
Can't insert mages

### ISSUE THREAD

**Issue by mauriziopassariello** [hackmdio/hackmd-io-issues#252]
### Is this a hackmd.io issue?

- [x ] Yes, the bug happens on https://hackmd.io
- [ ] No. If it's a self-hosted CodiMD instance, please fire up the issue in the repo of CodiMD: https://github.com/hackmdio/codimd

### Read our Official tutorial

It's on https://hackmd.io/c/tutorials/

或是中文版 https://hackmd.io/c/tutorials-tw/

### What's the problem
I don't seem able to insert images/pictures
#### Current behaviour
The image doesn't get uploaded.
> Screenshot (optional)

Provide steps to reproduce:

If possible, create a note with reproducible content and share the link with us.

e.g. note link: https://hackmd.io/features

#### Expected behaviour

(description of expected behaviour)

#### Environment

I use HackMD on:

**Desktop**
- mac
- Browser

**Mobile**

- Device: Android or Mac

**Comment by jackycute** [comment]
Hi @mauriziopassariello,
Can you still reproduce this issue?

**Comment by mauriziopassariello** [comment]
Yes still the sameOn 14 Apr 2021 11:48, Max Wu ***@***.***> wrote:
Hi @mauriziopassariello,
Can you still reproduce this issue?

**Comment by jackycute** [comment]
Unfortunately, we can't reproduce this issue from our side. Could you take a look at your dev tool console and see if there is any error? It will be even better if you can take screenshot or record a clip.
Thanks for helping us.

**Comment by jackycute** [comment]
Please also let us know which url you are browsing and what are the OS and browser versions?

**Comment by asucher24** [comment]
Not sure if its the same problem, but i'm also not able to insert images (from clipboard) on hackmd.io

For different browsers on ubuntu it works perfect, but for following it doesnt:
Devices: Android 11; (Smartphone and Tablet)
Browsers: Chrome 90.0.4430.82; Firefox 88.1.1; Startpage 5.80.0

**Comment by moskovich** [comment]
I have the same issue with Safari Version 14.1 (16611.1.21.161.6). I can use Chrome Version 90.0.4430.93 and it works as expected.

**Comment by taku08546** [comment]
Same issue as given above.
Chrome Version 90.0.4430.212

**Comment by Asienwald** [comment]
Having the same issue on both firefox 90.0 and chrome version 91.0.4472.124

Sometimes I have to paste it a few times (3-4 times) for it to finally register

console shows a 403 forbidden post request to imgur

https://user-images.githubusercontent.com/45307694/126046252-86a5c253-f55b-417d-a57b-29bd1aed1141.mp4
![403 hackmd](https://user-images.githubusercontent.com/45307694/126046333-87b52fee-0a7e-442d-a06a-e6c122fad5c5.png)

**Comment by Potherca** [comment]
I have a similar issue. With me, the cause was that my image was apparently too large.

![image](https://user-images.githubusercontent.com/195757/137332581-823a60a5-f2bd-4f09-8099-96e2fb011218.png)

It would be nice of the error could be displayed as a message in the UI.

By the way, I love [hackmd.io](https://hackmd.io/), keep up the great work! :hugs:

**Comment by Echsecutor** [comment]
I have a similar issue. 

Klicking the "Insert Image" Button brings up the OS file chooser dialog. Choosing any image there does not lead to any change in the document. Countrary to the other reported issues, I also do not see any network transaction.

Happened on Ubuntu 21.10 (Desktop)
- Chrome Version 97.0.4692.71 (Official Build) (64-bit)
as well as
- firefox 95.0.2 (64-bit)

I can not tell exactly when the behaviour started, but last year the image upload was working fine.

**Comment by dengwx11** [comment]
Having this weird issue on one of my macbooks. 

- Chrome Version 109.0.5414.87 (amd64) as well as  15.2

Have restarted or reinstalled my chrome but still cannot fix. But it works fine on the other macbook pro.
